drv/Function.drv

```diff
@@ -104,9 +159,13 @@ public interface FUNCTION KEY_VALUE_GENERIC extends Function<KEY_GENERIC_CLASS,
 
 	/** {@inheritDoc}
 	 * @deprecated Please use the corresponding type-specific method instead. */
+
 	@Deprecated
 	@Override
-	boolean containsKey(Object key);
+	default boolean containsKey(Object key) {
```

Comment:
So: the rationale for type-specific methods is that null is not part of the map (sorry, above should be "returning false"), but it is handled gracefully. Anything else get a OBJ2TYPE() and then it is handled as a primitive type.

Just check the previous code. Those methods were already implemented in the abstract classes, but you probably rewrote them instead of moving them.